docker/cygnus-ngsi/README.md

```diff
@@ -0,0 +1,190 @@
+#<a name="top"></a>cygnus-ngsi docker
+Content:
+
+* [Before starting](#section1)
+* [Getting an image](#section2)
+    * [Building form sources](#section2.1)
+    * [Using docker hub image](#section2.2)
+* [Using the image](#section3)
+    * [As it is](#section3.1)
+    * [Using a specific configuration](#section3.2)
+        * [Editing the docker files](#section3.2.1)
+        * [Environment variables](#section3.2.2)
+        * [Using volumnes](#section3.2.3)
+
+##<a name="section1"></a>Before starting
+Obviously, you will need docker installed and running in you machine. Please, check [this](https://docs.docker.com/linux/started/) official start guide.
+
+[Top](#top)
+
+##<a name="section2"></a>Getting an image
+###<a name="section2.1"></a>Building from sources
+Start by cloning the `fiware-cygnus` repository:
+
+    $ git clone https://github.com/telefonicaid/fiware-cygnus.git
+    
+Change directory to `docker/cygnus-ngsi`:
+
+    $ cd docker/cygnus-ngsi
+    
+And run the following command:
+
+    $ docker build -t cygnus-ngsi .
+    
+You can check the available images at your docker by typing:
+
+```
+$ docker images
+REPOSITORY          TAG                 IMAGE ID            CREATED              VIRTUAL SIZE
+cygnus-ngsi         latest              df556c817163        About a minute ago   1.132 GB
+centos              6                   61bf77ab8841        4 weeks ago          228.9 MB
+```
+
+[Top](#top)
+
+###<a name="section2.2"></a>Using docker hub image
+Instead of building an image from the scratch, you may download it from [hub.docker.com](docker pull fiware/cygnus):
+
+    $ docker pull fiware/cygnus-ngsi
+    
+It can be listed the same way than above:
+
+```
+$ docker images
+REPOSITORY          TAG                 IMAGE ID            CREATED              VIRTUAL SIZE
+cygnus-ngsi         latest              df556c817163        About a minute ago   1.132 GB
+centos              6                   61bf77ab8841        4 weeks ago          228.9 MB
+```
+
+[Top](#top)
+
+##<a name="section3"></a>Using the image
+###<a name="section3.1"></a>As it is
+The cygnus-ngsi image (either built from the scratch, either downloaded from hu.docker.com) allows running a Cuygnus agent in charge of receiving NGSI-like notifications and persiting them into a MySQL database running at `mysql` host.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/993/commits/59d14b3709656c1f7bde8e2d00776f79de6d1c0a